modules/version/ComponentsVersions.js

```diff
@@ -63,23 +63,19 @@ function(node, mucResource, mucJid) {
 
         var version = item.value;
         if (this.versions[componentName] !== version) {
-            if(this.versions[componentName] &&
```

Comment:
Will XMPP version(Prosody in our case) change always be incompatible ? Btw. why we did not compare if componentName === ComponentsVersions.XMPP_SERVER_COMPONENT in the first place ?